internal/cri/config/config.go

```diff
@@ -409,6 +409,9 @@ type RuntimeConfig struct {
 	// IgnoreDeprecationWarnings is the list of the deprecation IDs (such as "io.containerd.deprecation/pull-schema-1-image")
 	// that should be ignored for checking "ContainerdHasNoDeprecationWarnings" condition.
 	IgnoreDeprecationWarnings []string `toml:"ignore_deprecation_warnings" json:"ignoreDeprecationWarnings"`
+
+	// StatsMetricsPeriod is the period (in seconds) of containersm metrics stats collection.
```

Comment:
I guess this defaults to zero?  Should expand this comment to describe the behavior when it is zero verse when it is set and the Kubelet expectation (10s) although it can be any other value?